<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEditionTache</class>
 <widget class="QDialog" name="DialogEditionTache">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modifier la tâche</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Modifier la tâche</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxFormulaire">
     <property name="title">
      <string>Informations de la tâche</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_titre">
        <property name="text">
         <string>Titre * :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="input_titre">
        <property name="placeholderText">
         <string>Titre de la tâche</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_description">
        <property name="text">
         <string>Description :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPlainTextEdit" name="input_description">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Description de la tâche (optionnelle)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_date_debut">
        <property name="text">
         <string>Date de début :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateTimeEdit" name="input_date_debut">
        <property name="displayFormat">
         <string>dd/MM/yyyy HH:mm</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_date_fin">
        <property name="text">
         <string>Date de fin prévu:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDateTimeEdit" name="input_date_fin">
        <property name="displayFormat">
         <string>dd/MM/yyyy HH:mm</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_etat">
        <property name="text">
         <string>État *:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="input_etat">
        <item>
         <property name="text">
          <string>À faire</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>En cours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Réalisé</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Abandonné</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>En attente</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_info">
     <property name="styleSheet">
      <string notr="true">font-style: italic; color: #888;</string>
     </property>
     <property name="text">
      <string>* Champs obligatoires</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnValider">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Valider les modifications</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAnnuler">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
